moml/data/data_loader.py

```
@@ -1,305 +1,431 @@
 
-from __future__ import annotations
+Unified PFAS data loader for molecular machine learning applications.
+
+This module provides the PFASDataLoader class which integrates molecular graph
+processing utilities with optional quantum mechanical features and environmental
+context information. It returns PyTorch Geometric Data objects ready for
+training MGNN models.
+"""
 
-import os
 import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)